### DISCUSSION TITLE
[Bug]: `KafkaConnect` default network policy

### DISCUSSION THREAD

**Discussion by dermasmid** [strimzi/strimzi-kafka-operator#10186] [General]
### Bug Description

when creating a restrictive `networkpolicy` by default, and i didnt find a way to configure it

### Strimzi version

0.41.0

### Kubernetes version

1.27.8-gke.1067004

### Installation method

yaml files

### Infrastructure

GKe

**Comment by scholzj** [comment]
Maybe you should start by explaining what exactly is it you see as a problem, what is your configuration etc.
